indico/web/templates/forms/palette_picker_widget.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'forms/base_widget.html' %}
+
+{% block html %}
+    {% set colors = field.data._asdict() %}
```

Comment:
Usually you call `field._value()` to get the field value as used in the widget. So you could return the dict from that method